src/vaishnaviVJun20_arrayExamples/MeetingRoom.java

```diff
@@ -0,0 +1,93 @@
+c. Write a method which helps to book a meeting room by giving the name of the meeting room and given hrs.
+d. displayInfo() method should print booked meeting room info [i.e. meeting room,
+booked for how many hrs].
+e. displayAvilableMeetingRoomForBooking() method will print available meeting room names.
+[i.e. If Nalanda and Takshashila is booked then this method should print "Available meeting rooms: AgraFort and
+PratapGadh"]
+*/
+
+import java.util.Scanner;
+
+public class MeetingRoom {
+
+	int nalandaCount, takshashilaCount, agraFortCount, pratapGadhCount;
+
+	void bookMeetingRoom() {
+		nalandaCount++;
+		displayInfo("Nalanda", 1);
+	}
+
+	void bookMeetingRoom(String meetingRoomName) {
+
+		if (meetingRoomName.equals("Takshashila")) {
+			takshashilaCount++;
+			displayInfo(meetingRoomName, 1);
+		} else if (meetingRoomName.equals("AgraFort")) {
+			agraFortCount++;
+			displayInfo(meetingRoomName, 1);
+		} else if (meetingRoomName.equals("PratapGadh")) {
+			pratapGadhCount++;
+			displayInfo(meetingRoomName, 1);
+		} else
+			System.out.println("Inavlid Entry");
+	}
+
+	void bookMeetingRoom(String meetingRoomName, int hrs) {
+
+		if (meetingRoomName.equals("Takshashila")) {
+			takshashilaCount++;
+			displayInfo(meetingRoomName, hrs);
+		} else if (meetingRoomName.equals("AgraFort")) {
+			agraFortCount++;
+			displayInfo(meetingRoomName, hrs);
+		} else if (meetingRoomName.equals("PratapGadh")) {
+			pratapGadhCount++;
+			displayInfo(meetingRoomName, hrs);
+		} else if (meetingRoomName.equals("Nalanda")) {
+			nalandaCount++;
+			displayInfo(meetingRoomName, hrs);
+		} else
+			System.out.println("Inavlid Entry");
+
+	}
+
+	void displayInfo(String meetingRoomName, int hrs) {
+		System.out.println("You have booked " + meetingRoomName + " for " + hrs + " hour");
+	}
+
+	void displayAvilableMeetingRoomForBooking() {
+		if (nalandaCount == 0)
+			System.out.println("Nalanda is available for booking");
+		if (takshashilaCount == 0)
+			System.out.println("Takshashila is available for booking");
+		if (agraFortCount == 0)
+			System.out.println("AgraFort is available for booking");
+		if (pratapGadhCount == 0)
+			System.out.println("PratapGadh is available for booking");
+	}
+
+	public static void main(String[] args) {
+		MeetingRoom meetingRoom = new MeetingRoom();
+		Scanner scanner = new Scanner(System.in);
+		meetingRoom.bookMeetingRoom();
+		System.out.println("Which Meeting room would you like to book : Takshashila/AgraFort/PratapGadh ? ");
+		String meetingRoomName = scanner.next();
+		meetingRoom.bookMeetingRoom(meetingRoomName);
+		System.out.println("Which Meeting room would you like to book and for how many hours : Takshashila/AgraFort/PratapGadh/Nalanda ? ");
+		meetingRoomName = scanner.next();
+		int hrs = scanner.nextInt();
+		meetingRoom.bookMeetingRoom(meetingRoomName, hrs);
+		meetingRoom.displayAvilableMeetingRoomForBooking();
+		scanner.close();
```

Comment:
Hi @VaishnaviVJun20,
Here while calling method you are printing time for booked room but you need to check which for room booked for how many hours, this FR not fulfilled in code

what you should do is define String array named as meetingRoomName and second array which contains hours.
when someone booked room that time you should update the same index value.

String[] meetingRoomName = {"Nalanda", "Takshashila", "AgraFort","PratapGadh"};
int hours[] = {0,0,0,0};

this hint may help you to think on this. for now we know array which can store multiple value, so think on this

Reply:
-- Updated the code, Please review